numba/cuda/tests/cudadrv/test_cuda_devicerecord.py

```diff
@@ -85,6 +86,7 @@ class TestCudaDeviceRecordWithRecord(TestCudaDeviceRecord):
     Tests the DeviceRecord class with np.record host types
     """
     def setUp(self):
+        CUDATestCase.setUp()
```

Comment:
Missing `self`. Should be `super().setUp()`

Reply:
This will call `TestCudaDeviceRecord.setUp()`, which is what I was trying to avoid with the suggestion to use `CUDATestCase.setUp()` instead.

Reply:
In that case, it needs to be `CUDATestCase.setUp(self)`

Reply:
Example of errors:

```pytb
======================================================================
ERROR: test_auto_device (numba.cuda.tests.cudadrv.test_cuda_devicerecord.TestCudaDeviceRecordWithRecord)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/siu/dev/numba/numba/cuda/tests/cudadrv/test_cuda_devicerecord.py", line 89, in setUp
    CUDATestCase.setUp()
TypeError: setUp() missing 1 required positional argument: 'self'

======================================================================
ERROR: test_device_record_copy (numba.cuda.tests.cudadrv.test_cuda_devicerecord.TestCudaDeviceRecordWithRecord)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/siu/dev/numba/numba/cuda/tests/cudadrv/test_cuda_devicerecord.py", line 89, in setUp
    CUDATestCase.setUp()
TypeError: setUp() missing 1 required positional argument: 'self'

======================================================================
ERROR: test_device_record_interface (numba.cuda.tests.cudadrv.test_cuda_devicerecord.TestCudaDeviceRecordWithRecord)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/siu/dev/numba/numba/cuda/tests/cudadrv/test_cuda_devicerecord.py", line 89, in setUp
    CUDATestCase.setUp()
TypeError: setUp() missing 1 required positional argument: 'self'

======================================================================
ERROR: test_from_record_like (numba.cuda.tests.cudadrv.test_cuda_devicerecord.TestCudaDeviceRecordWithRecord)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/siu/dev/numba/numba/cuda/tests/cudadrv/test_cuda_devicerecord.py", line 89, in setUp
    CUDATestCase.setUp()
TypeError: setUp() missing 1 required positional argument: 'self'

----------------------------------------------------------------------
```

Reply:
Apologies for missing `self` in my earlier suggestion - I thought I tested my suggestions locally, but I must have done something not quite right!